js/layout-javascript/news-feed-code.js

```diff
@@ -144,6 +144,20 @@ DynamicList.prototype.attachObservers = function() {
   });
 
   _this.$container
+    .on('show.bs.dropdown', function(e) {
+      var parentAccordion = $(e.currentTarget).parents('[data-collapse-id]')[0];
```

Comment:
https://github.com/airbnb/javascript/blob/master/README.md#jquery--dollar-prefix

```suggestion
      $(e.currentTarget).parents('[data-collapse-id]').style('overflow', 'visible');
```

Why did you choose to use `e.currentTarget` instead of `e.target`?

Reply:
Because `e.currentTarget` is child of needed DOM element, but now I understand that these makes no sense when using `.parents()` function.
Fixed and added fix to prevent overlapping the dropdown:
![image](https://user-images.githubusercontent.com/74735207/107066154-47d1c180-67e6-11eb-968b-0dd54e0ffc63.png)